packages/storybook/src/nimble/table-column/menu-button/table-column-menu-button.mdx

```diff
@@ -32,3 +33,23 @@ responsibility to do the following:
    `newState` is `true`.
 2. Update the data through a `setData()` call when a menu item is activated in
    order to update the text on the menu button in the column.
+
+### Menu Button Column vs Table Action Menu
+
+Both an action menu on a column and the <Tag name={tableColumnMenuButtonTag}/>
+can be used to provide a menu associated with a row in the table. However, they
+are used in different contexts.
+
+An action menu should be used in the table when:
+
+-   the items in the menu are actions to be performed on the record
+-   selecting an item in the menu does not change the value of the record
+
+The <Tag name={tableColumnMenuButtonTag}/> should be used when:
+
+-   the items in the menu represent values that can be assigned to the record
+-   the currently selected value should be displayed in the table
+
+When using the <Tag name={tableColumnMenuButtonTag}/>, the currently selected
```

Comment:
I think this note would fit better as a bullet under the list above ("it is the client's responsibility to do the following")